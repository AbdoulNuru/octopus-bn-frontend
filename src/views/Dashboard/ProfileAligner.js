import React from "react";
import classes from '../../styles/profileAligner.module.css';

const ProfileAligner = (props) => {
    return(
    <div className={classes.SvgDiv}>
        <div className={classes.SideIntro}>

        </div>
    </div>

    );
}

export default ProfileAligner;
