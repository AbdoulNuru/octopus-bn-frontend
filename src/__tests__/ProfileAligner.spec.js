import React, { Component } from 'react';
import { render} from '@testing-library/react';
import ProfileAligner from '../views/Dashboard/ProfileAligner';


const ProfileAlignerComponent = () => {
	return render(

	   <ProfileAligner />
	);
};
describe('Profile Component', () => {
	it('should render profile aligner properly', () => {
		const { asFragment } = ProfileAlignerComponent();
		expect(asFragment(<ProfileAligner/>)).toMatchSnapshot();
    });
});
