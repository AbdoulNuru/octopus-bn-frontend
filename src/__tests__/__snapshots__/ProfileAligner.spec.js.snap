// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Component should render profile aligner properly 1`] = `
<DocumentFragment>
  <div>
    <div />
  </div>
</DocumentFragment>
`;
